plugins/drude/openmmapi/src/DrudeHelpers.cpp

```diff
@@ -127,12 +127,12 @@ vector<Vec3> assignDrudeVelocities(const System &system, double temperature, dou
     return velocities;
 }
 
-double computeSystemTemperatureFromVelocities(const System& system, const vector<Vec3>& velocities) {
+pair<double, double> computeSystemTemperatureFromVelocities(const System& system, const vector<Vec3>& velocities) {
```

Comment:
The function name is no longer accurate.  Maybe rename it to `computeTemperaturesFromVelocities()`, and add a comment above describing what it computes?

Reply:
done